sql-plugin/src/main/311until320-all/scala/com/nvidia/spark/rapids/shims/ParquetSchemaClipShims.scala

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.shims
+
+import org.apache.parquet.schema._
+import org.apache.parquet.schema.OriginalType._
+import org.apache.parquet.schema.PrimitiveType.PrimitiveTypeName._
+
+import org.apache.spark.sql.internal.SQLConf
+import org.apache.spark.sql.rapids.execution.TrampolineUtil
+import org.apache.spark.sql.types._
+
+object ParquetSchemaClipShims {
+  /** Stubs for configs not defined before Spark 330 */
+  def useFieldId(conf: SQLConf): Boolean = false
+
+  def ignoreMissingIds(conf: SQLConf): Boolean = false
+
+  def timestampNTZEnabled(conf: SQLConf): Boolean = false
```

Comment:
Seems this is no longer used. Beter to remove it.

Reply:
Removed this and the corresponding method in other shims.